guides/common/modules/con_dhcp-service-providers.adoc

```diff
@@ -0,0 +1,27 @@
+[id="dhcp-serivce-proviers"]
+= DHCP service providers
+
+{Project} can manage IP leases on a DHCP server through a {SmartProxy}.
+This management contains querying for available IP addresses, adding new reservations, and deleting existing reservations from the lease database.
+Note that {SmartProxy} cannot manage subnet declarations.
+
+{SmartProxy} supports the following DHCP providers that you can use to integrate {Project} with your existing DHCP infrastructure or deploy a new one:
+
+`dhcp_isc`:: Managing IP leases on an ISC dhcpd server by using the Object Management Application Programming Interface (OMAPI).
```

Comment:
Should dhcpd have some markup? You're using markup in at least one another module:
```suggestion
`dhcp_isc`:: Managing IP leases on an ISC `dhcpd` server by using the Object Management Application Programming Interface (OMAPI).
```
Applies for the dlist item below as well.

Reply:
You know what, actually, here you are writing "ISC dhcpd" but at least in one of the section headings it's "ISC DHCP". And line 21 has "dnsmasq DHCP". Should this be consistent? (And there are other occurrences of `dhcpd server` vs `DHCP server` in case you want to try to unify them throughout the doc.)

Reply:
ISC DHCP is the project name, which is really confusing but I'm pretty sure it dates back to a time when there weren't that many alternatives.